spec/controllers/idv/phone_controller_spec.rb

```diff
@@ -157,16 +158,14 @@
       it 'tracks form error and does not make a vendor API call' do
         expect_any_instance_of(Idv::Agent).to_not receive(:proof_address)
 
-        put :create, params: { idv_phone_form: { phone: '703' } }
+        put :create, params: { idv_phone_form: { phone: short_phone } }
 
```

Comment:
I like this pattern of changing the phone number from a hard coded value to a variable named to show what kind of entry we are testing. This is easier to maintain as the value only needs to be updated in one place if it would ever need to change. Nice :)

Reply:
🙌